lib/fibonacci.rb

```diff
@@ -1,8 +1,19 @@
 # Improved Fibonacci
 
-# Time Complexity - ?
-# Space Complexity - ?  (should be O(n))
+# Time Complexity - O(1)
+# Space Complexity - O(n)  (should be O(n))
 # Hint, you may want a recursive helper method
 def fibonacci(n)
```

Comment:
Time complexity is O(1)????  You mean it won't take longer if n gets bigger... Actually this solution is O(n) for both time and space complexity.